packages/contractkit/src/providers/celo-private-keys-subprovider.ts

```diff
@@ -124,7 +124,7 @@ export class CeloPrivateKeysWalletProvider extends PrivateKeyWalletSubprovider {
       txParams.gatewayFeeRecipient = await this.getCoinbase()
     }
     if (!isEmpty(txParams.gatewayFeeRecipient) && isEmpty(txParams.gatewayFee)) {
-      txParams.gatewayFee = DefaultGatewayFee.toString()
+      txParams.gatewayFee = DefaultGatewayFee.toString(16)
```

Comment:
It seems like this should be prefixed with '0x'. If it works without it though, I guess I am mistaken.
```suggestion
      txParams.gatewayFee = '0x' + DefaultGatewayFee.toString(16)
```

Reply:
So it works because of this very confusing implmentation

https://github.com/MaiaVictor/eth-lib/blob/master/src/bytes.js#L77

Basically, `fromNat`/`rlp.encode` accepts any value that is a string and has an even number of chars. I figure we want to change this behavior and make it safer, but did not want to dig deeper right now